Tests/ReactiveSwiftTests/LifetimeSpec.swift

```diff
@@ -1,34 +1,97 @@
 import Quick
 import Nimble
-import ReactiveSwift
+@testable import ReactiveSwift
 import Result
 
 final class LifetimeSpec: QuickSpec {
 	override func spec() {
 		describe("Lifetime") {
-			it("should complete its lifetime ended signal when the it deinitializes") {
-				let object = MutableReference(TestObject())
+			describe("ended") {
```

Comment:
It's a nitpick, but do you think we can make this change without reindenting the file? It makes the diff a little hard to follow, and there's no shared setup/teardown code.